### ISSUE TITLE
Access panels on record view - small details

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#342]
Add `<h2 class="sr-only">Access</h2>` at the top of the column.

Panel heading text should be h3:  

```
<div class="panel-heading">
      <h3>Online</h3>
  </div>
```

Set alt="" for all library images in panel headings (the image is only decorative and the text is redundant)
